com.github.dynobo.sphereview.yml

```diff
@@ -0,0 +1,67 @@
+app-id: com.github.dynobo.sphereview
```

Comment:
the app-id doesn't follow the requirements. 

it is supposed to be starting with `io.github`

Reply:
Is this a new requirement? For my last app (`com.github.dynobo.normcap`) this was the way to go...

Reply:
Fine, I'll change that, too. (Still wondering, when this rule change...)

Reply:
Done in https://github.com/dynobo/sphereview/commit/73f918edb52d7c8340d0e26798cdac4cbb9ef31d and https://github.com/flathub/flathub/pull/6675/commits/2e59bcf009e2ed7e1fcbe3d361d0bdc18eaa6dae